domehunter/dome_control.py

```diff
@@ -305,7 +334,12 @@ def abort(self):
         # one way might be cut power to the automationHAT so the motor relays
         # will receive no voltage even if the relay is in the open position?
         logger.warning(f'Aborting dome movement')
-        self._stop_moving()
+        # set the abort event thread flag
+        self._abort_event.set()
+        # wait for threads to abort
+        while self._move_event.is_set():
+            pass
```

Comment:
Always good to have some minor pause.
```suggestion
            time.sleep(0.1)
```